src/resolvers/projectResolver.test.ts

```diff
@@ -1262,6 +1277,145 @@ function allProjectsTestCases() {
       ),
     );
   });
+  it('should return projects, filter by accept donation on Solana', async () => {
+    const savedProject = await saveProjectDirectlyToDb({
+      ...createProjectData(),
+      title: String(new Date().getTime()),
+      slug: String(new Date().getTime()),
+    });
+    await ProjectAddress.delete({ projectId: savedProject.id });
+    const solanaAddress = ProjectAddress.create({
+      project: savedProject,
+      title: 'first address',
+      address: generateRandomSolanaAddress(),
+      chainType: ChainType.SOLANA,
+      networkId: 0,
+      isRecipient: true,
+    });
+    await solanaAddress.save();
+    const result = await axios.post(graphqlUrl, {
+      query: fetchMultiFilterAllProjectsQuery,
+      variables: {
+        filters: ['AcceptFundOnSolana'],
+        sortingBy: SortingField.Newest,
+      },
+    });
+    result.data.data.allProjects.projects.forEach(project => {
+      assert.isOk(
+        project.addresses.find(
+          address =>
+            address.isRecipient === true &&
+            address.chainType === ChainType.SOLANA,
+        ),
+      );
+    });
+    assert.isOk(
+      result.data.data.allProjects.projects.find(
+        project => Number(project.id) === Number(savedProject.id),
+      ),
+    );
+  });
+  it('should return projects, filter by accept fund on two Ethereum networks', async () => {
+    const project = await saveProjectDirectlyToDb({
+      ...createProjectData(),
+      title: String(new Date().getTime()),
+      slug: String(new Date().getTime()),
+    });
+
+    const mainnetAddress = ProjectAddress.create({
+      project,
+      title: 'first address',
+      address: generateRandomEtheriumAddress(),
+      networkId: 1,
+      isRecipient: true,
+    });
+    await mainnetAddress.save();
+
+    const solanaAddress = ProjectAddress.create({
+      project,
+      title: 'secnod address',
+      address: generateRandomSolanaAddress(),
+      chainType: ChainType.SOLANA,
+      networkId: 0,
+      isRecipient: true,
+    });
+    await solanaAddress.save();
+
+    const result = await axios.post(graphqlUrl, {
+      query: fetchMultiFilterAllProjectsQuery,
+      variables: {
+        filters: ['AcceptFundOnMainnet', 'AcceptFundOnSolana'],
```

Comment:
@CarlosQ96 Look at this test, all other test scenarios we have had before only test filters with single critrion.